nbs/13a_learner.ipynb

```diff
@@ -24,7 +24,20 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/opt/conda/lib/python3.9/site-packages/torch/cuda/__init__.py:145: UserWarning: \n",
```

Comment:
Can you fix this? It'll mess up the docs

Reply:
Ack, yup sorry! Will do so momentarily